<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="AHS-LP-181" failures="0" tests="6" name="utility.testNgdemo1" time="23.497" errors="1" timestamp="2022-04-30T00:18:07 IST" skipped="0">
  <testcase classname="utility.testNgdemo1" name="enterCredentials" time="10.048"/>
  <system-out/>
  <testcase classname="utility.testNgdemo1" name="enterCredentials" time="2.915"/>
  <system-out/>
  <testcase classname="utility.testNgdemo1" name="generateReports" time="0.911">
    <error message="Failed to open file:/C:/Users/Naresh/git/TestNG-Repo/TestNGProject/test-output/ExtentReports/Default%20suite-30-17-2022%2012-17-32.html. Error message: The system cannot find the file specified.
" type="java.io.IOException">
      <![CDATA[java.io.IOException: Failed to open file:/C:/Users/Naresh/git/TestNG-Repo/TestNGProject/test-output/ExtentReports/Default%20suite-30-17-2022%2012-17-32.html. Error message: The system cannot find the file specified.

at java.desktop/sun.awt.windows.WDesktopPeer.ShellExecute(WDesktopPeer.java:115)
at java.desktop/sun.awt.windows.WDesktopPeer.browse(WDesktopPeer.java:101)
at java.desktop/java.awt.Desktop.browse(Desktop.java:533)
at utility.Extentreports_Base.generateReports(Extentreports_Base.java:70)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:65)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:381)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:319)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:396)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- generateReports -->
  <system-out/>
  <testcase classname="utility.testNgdemo1" name="enterCredentials" time="3.056"/>
  <system-out/>
  <testcase classname="utility.testNgdemo1" name="enterCredentials" time="3.787"/>
  <system-out/>
  <testcase classname="utility.testNgdemo1" name="enterCredentials" time="2.780"/>
  <system-out/>
</testsuite> <!-- utility.testNgdemo1 -->
